pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/HashJoinOperator.java

```diff
@@ -307,7 +303,7 @@ private TransferableBlock buildJoinedDataBlock() {
     }
   }
 
-  private List<Object[]> buildJoinedRows(TransferableBlock leftBlock) {
+  private List<Object[]> buildJoinedRows(TransferableBlock leftBlock) throws ProcessingException {
```

Comment:
(format) Please apply Pinot Style (exception should be on the next line)

Reply:
My bad, created this minor fix PR - https://github.com/apache/pinot/pull/13933